fhir-server-test/src/test/java/org/linuxforhealth/fhir/server/test/BundleTest.java

```diff
@@ -1542,51 +1546,58 @@ public void testTransactionMixture() throws Exception {
-        // search
-//      bundle = addRequestToBundle(null, bundle, HTTPVerb.GET, "Patient?family=Ortiz&_count=100", null, null);
-
-        printBundle(method, "request", bundle);
-
-        Entity<Bundle> entity = Entity.entity(bundle, FHIRMediaType.APPLICATION_FHIR_JSON);
-        Response response = target.request().post(entity, Response.class);
-        assertResponse(response, Response.Status.OK.getStatusCode());
-        Bundle responseBundle = getEntityWithExtraWork(response,method);
-        assertResponseBundle(responseBundle, BundleType.TRANSACTION_RESPONSE, 4);
-        assertGoodPostPutResponse(responseBundle.getEntry().get(0), Status.CREATED.getStatusCode());
-        assertGoodPostPutResponse(responseBundle.getEntry().get(1), Status.OK.getStatusCode());
-        assertGoodGetResponse(responseBundle.getEntry().get(2), Status.OK.getStatusCode());
-        assertGoodGetResponse(responseBundle.getEntry().get(3), Status.OK.getStatusCode());
-//      assertGoodGetResponse(responseBundle.getEntry().get(4), Status.OK.getStatusCode());
-//      assertGoodGetResponse(responseBundle.getEntry().get(5), Status.OK.getStatusCode());
-
-//      Bundle resultSet;
-
-        // Verify the history results.
-//      resultSet = (Bundle) FHIRUtil.getResourceContainerResource(responseBundle.getEntry().get(4).getResource());
-//      assertNotNull(resultSet);
-//      assertTrue(resultSet.getEntry().size() > 2);
-
-        // Verify the search results.
-//      resultSet = (Bundle) FHIRUtil.getResourceContainerResource(responseBundle.getEntry().get(5).getResource());
-//      assertNotNull(resultSet);
-//      assertTrue(resultSet.getEntry().size() > 3);
+        
+        List<String> preferredHeaders = List.of(PREFER_HEADER_RETURN_REPRESENTATION, PREFER_HEADER_RETURN_OPERATION_OUTCOME);
+        for (String preferredHeader : preferredHeaders) { 
+            WebTarget target = getWebTarget();
+
+            // Perform a mixture of request types.
+            Bundle bundle = buildBundle(BundleType.TRANSACTION);
+            // create
+            bundle = addRequestToBundle(null, bundle, HTTPVerb.POST, "Patient", null,
+                    TestUtil.readLocalResource("Patient_DavidOrtiz.json"));
+            // update
+            bundle = addRequestToBundle(null, bundle, HTTPVerb.PUT, "Patient/" + patientT1.getId(), null,
+                    patientT1);
+            // read
+            bundle = addRequestToBundle(null, bundle, HTTPVerb.GET, "Patient/" + patientT2.getId(), null, null);
+            // vread
+            bundle = addRequestToBundle(null, bundle, HTTPVerb.GET, locationT1, null, null);
+            // history
+//          bundle = addRequestToBundle(null, bundle, HTTPVerb.GET, "Patient/" + patientT1.getId() + "/_history", null, null);
+            // search
+//          bundle = addRequestToBundle(null, bundle, HTTPVerb.GET, "Patient?family=Ortiz&_count=100", null, null);
```

Comment:
I see the original test had these commented out too, but it makes me wonder why...